hssqlppp/tests/Database/HsSqlPpp/Tests/Parsing/FunctionsDdl.lhs

```diff
@@ -8,11 +8,16 @@
 > import Database.HsSqlPpp.Tests.Parsing.Utils
 > import Database.HsSqlPpp.Tests.TestTypes
 
-> functionsDdl:: Item
+> functionsDdl :: Item
 > functionsDdl =
 >    Group "functionsddl" [
 >      Group "basics" [
->       s "create function t1(text) returns text as $$\n\
+>       s "create function t1(x int) returns int as $$ return x+2 $$ language python;"
+>       [CreateFunction ea (name "t1") [ParamDef ea (Nmc "x")  $ st "int"]
+>        (st "int") NoReplace Python
+>        (PythonFnBody " return x+2 ")
```

Comment:
any reason this is not `"return x+2"` (without spaces before and after)?

Reply:
Also maybe let's also add a `create or replace` case just to make sure that it works.

Reply:
the spaces are because I wrote the query with spaces